pages/clustering/high-availability.mdx

```diff
@@ -138,8 +139,9 @@ export MEMGRAPH_EXPERIMENTAL_ENABLED=high-availability
 export MEMGRAPH_COORDINATOR_PORT=10111
 export MEMGRAPH_COORDINATOR_ID=1
 export MEMGRAPH_BOLT_PORT=7692
+export MEMGRAPH_NURAFT_LOG_FILE="<path-to-log-file>"
```

Comment:
Not sure, but should the envs all be documented at /docs/database-management/configuration#environment-variables as well?

Reply:
I've created an issue for - > this https://github.com/memgraph/documentation/issues/874